### ISSUE TITLE
Font URLs/paths do not resolve in Angular/Angular CLI on Windows

### ISSUE THREAD

**Issue by EatonZ** [IBM/plex#182]
If you create a new Angular project using Angular CLI **on Windows** and then add this path to `styles` in `angular.json`: `node_modules/@ibm/plex/css/ibm-plex.css`

..it fails to resolve font files:
![2018-07-30_15-05-33](https://user-images.githubusercontent.com/3020941/43418038-c2828db8-940a-11e8-8c52-dcd0b7eef800.png)

Attached below is a sample project you can try compiling. You should see what I mean right away.
Please note that the `node_modules/` path format works for all other stylesheet files I have ever needed to reference. I believe the problem is the way Windows, Angular, or Angular CLI resolves the `url`s in `ibm-plex.css`.

[ibm-sample.zip](https://github.com/IBM/plex/files/2242555/ibm-sample.zip)

**Comment by asudoh** [comment]
@EatonZ Thank you for reporting! I tried to run `ng new`, apply below patch and run `npm i`/`ng serve` (in my Windows7 machine), but couldn't reproduce the error - Instead, I can see Plex CSS correctly in DOM inspector.

```
diff -urw my-ibm-plex-app.orig/angular.json my-ibm-plex-app/angular.json
--- my-ibm-plex-app.orig/angular.json	2018-08-13 16:46:52.452531500 +0900
+++ my-ibm-plex-app/angular.json	2018-08-13 16:39:36.895619000 +0900
@@ -23,7 +23,8 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.css"
+              "src/styles.css",
+              "node_modules/@ibm/plex/css/ibm-plex.css"
             ],
             "scripts": []
           },
diff -urw my-ibm-plex-app.orig/package.json my-ibm-plex-app/package.json
--- my-ibm-plex-app.orig/package.json	2018-08-13 16:46:52.458531800 +0900
+++ my-ibm-plex-app/package.json	2018-08-13 16:42:53.463862100 +0900
@@ -29,6 +29,7 @@
     "@angular/cli": "~6.1.3",
     "@angular/compiler-cli": "^6.1.0",
     "@angular/language-service": "^6.1.0",
+    "@ibm/plex": "^1.1.3",
     "@types/jasmine": "~2.8.6",
     "@types/jasminewd2": "~2.0.3",
     "@types/node": "~8.9.4",
```

OTOH, it seems that your `.zip` has more changes than what `ng new` generates plus one-line change to `angular.json` you asserted. Would you try something similar to what I did, which would create more reduced case? Thanks!

**Comment by EatonZ** [comment]
@asudoh Sorry for the late response. I did end up figuring it out.

I didn't want to have the fonts in my site root, which was why the errors were occurring. I wanted them in a subfolder, so it was obvious I needed to use the scss instead to use a custom font-prefix. Using the css appears to mandate fonts in the root of the site, and you'll need to copy them into your project yourself.

It wasn't clear to me that I had to copy the fonts into my project manually, so I had to set that up through angular.json:
```
{
  "glob": "**/*.woff",
  "input": "node_modules/@ibm/plex",
  "output": "assets/fonts"
},
{
  "glob": "**/*.woff2",
  "input": "node_modules/@ibm/plex",
  "output": "assets/fonts"
}
```

I have everything set up to my satisfaction so I'm closing this as I don't think there is a bug here.
